dbms/tests/integration/test_dictionary_ddl_on_cluster/test.py

```diff
@@ -55,6 +56,16 @@ def test_dictionary_ddl_on_cluster(started_cluster):
         assert node.query("SELECT count() from sometbl") == "1\n"
         assert node.query("SELECT dictGetString('default.somedict', 'value', toUInt64({}))".format(num)) == node.name + '\n'
 
+
+    for num, node in enumerate([ch1, ch2, ch3, ch4]):
+        node.query("ALTER TABLE sometbl UPDATE value = 'new_key' WHERE 1")
+
+    ch1.query("SYSTEM RELOAD DICTIONARY ON CLUSTER 'cluster' `default.somedict`")
+    time.sleep(2)  # SYSTEM RELOAD DICTIONARY is an asynchronous query
```

Comment:
it's not already, since #8240, so `sleep` can be dropped

Reply:
b17048f07a